thirdparty/README.md

```diff
@@ -1,5 +1,12 @@
 # Third party libraries
 
+## assimp
+
+- Upstream: https://github.com/assimp
+- Version: git (933fe5eb43ff665807c180c245df7969642987cd) with modifications
```

Comment:
What are those modifications? We should not modify thirdparty code unless we **really** need to. And if we do, we should attempt to upstream our changes. If we do end up making custom changes, they need to be properly identified and done in a separate commit (with a patch added to the repo so that we can reapply them in the future).

Moreover, it would be much better to follow a tagged upstream release (like 4.1.0, or if we really need to go unstable, 5.0.0.rc1). Linux distros should be able to build Godot with their own system provided Assimp package (with the `builtin_assimp=no` SCons flag). If it's not possible right now, it's not too bad, but once 5.0.0 is released we should stick to it (and again, without custom changes).

Reply:
This is old. I believe the modifications were stripped out and undone when we shrunk the codebase.